app/src/main/java/net/skyscanner/backpack/demo/compose/NavBarStory.kt

```diff
@@ -146,6 +146,49 @@ fun TransparentNavBarStory(
     }
 }
 
+@Composable
+@NavBarComponent
+@ComposeStory("OnDark")
+fun OnDarkNavBarStory(
```

Comment:
NavBarStoryOnDark? Just for consistency

Reply:
Hmm, the story for the transparent one is named `TransparentNavBarStory`, that's why I named this one `OnDarkNavBarStory ` (`SurfaceContrastNavBarStory ` now). WDYT?

Reply:
Yeah, that makes sense. Marking this as resolved